templates/3a_monitor_MIBI_run.ipynb

```diff
@@ -1,12 +1,5 @@
 {
```

Comment:
<p>The additional arguments to qc_metrics don't need to be summarized for the users. Can you link each callback to the notebook that it corresponds to? That way if the user has a question for what it does they know where to look</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>